core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java

```diff
@@ -0,0 +1,787 @@
+        final Function function = new Function(FUNC_SYMBOL,
+                Arrays.<Type>asList(),
+                Arrays.<TypeReference<?>>asList(new TypeReference<Utf8String>() {}));
+        return executeRemoteCallSingleValueReturn(function, String.class);
+    }
+
+    /**
+     * Instance method for calling method "totalSupply" in KIP-7 Standard Token
+     *
+     * @return RemoteCall
+     */
+    public RemoteCall<BigInteger> totalSupply() {
+        final Function function = new Function(FUNC_TOTALSUPPLY,
+                Arrays.<Type>asList(),
+                Arrays.<TypeReference<?>>asList(new TypeReference<Uint256>() {}));
+        return executeRemoteCallSingleValueReturn(function, BigInteger.class);
+    }
+
+    /**
+     * Instance method for calling method "transfer" in KIP-7 Standard Token
+     * @param to The owner will receive the tokens.
+     * @param value The token amount will be transferred.
+     * @return RemoteCall
+     */
+    public RemoteCall<KlayTransactionReceipt.TransactionReceipt> transfer(String to, BigInteger value) {
+        final Function function = new Function(
+                FUNC_TRANSFER,
+                Arrays.<Type>asList(new org.web3j.abi.datatypes.Address(to),
+                        new org.web3j.abi.datatypes.generated.Uint256(value)),
+                Collections.<TypeReference<?>>emptyList());
+        return executeRemoteCallTransaction(function);
+    }
+
+    /**
+     * Instance method for calling method "transfer" in KIP-7 Standard Token
+     * @param from The current owner of the tokens.
+     * @param to The owner will receive the tokens.
+     * @param value The token amount will be transferred.
+     * @return RemoteCall
+     */
+    public RemoteCall<KlayTransactionReceipt.TransactionReceipt> transferFrom(String from, String to, BigInteger value) {
+        final Function function = new Function(
+                FUNC_TRANSFERFROM,
+                Arrays.<Type>asList(new org.web3j.abi.datatypes.Address(from),
+                        new org.web3j.abi.datatypes.Address(to),
+                        new org.web3j.abi.datatypes.generated.Uint256(value)),
+                Collections.<TypeReference<?>>emptyList());
+        return executeRemoteCallTransaction(function);
+    }
+
+    /**
+     * Instance method for calling method "unpause" in KIP-7 Standard Token
+     *
+     * @return RemoteCall
+     */
+    public RemoteCall<KlayTransactionReceipt.TransactionReceipt> unpause() {
+        final Function function = new Function(
+                FUNC_UNPAUSE,
+                Arrays.<Type>asList(),
+                Collections.<TypeReference<?>>emptyList());
+        return executeRemoteCallTransaction(function);
+    }
+
+    /**
+     * Static method create KIP7 Standard Token handler connecting existed Contract
+     * @param contractAddress KIP7 Standard Token Address
+     * @param caver Caver Object
+     * @param credentials KlayCredential Object to control Contract
```

Comment:
How about adding description for KIP7 will use this credential as a default credential?
The load method with transactionManager also, the default credential in transactionManager will use as a default credential in here.

Reply:
Okay, I'll comment it.

Reply:
I added comment. Would you check it?